acceptable/tests/test_main.py

```diff
@@ -612,10 +577,19 @@ def test_render_cmd_with_documentation_builder(self):
      ' Documentation: API foo at response_schema.foo_result.introduced_at'),
 ]
 
-EXPECTED_OPENAPI_RESULT = [
-    "null\n",
-    "...\n",
-]
+EXPECTED_OPENAPI_RESULT = """components_schemas: {}
```

Comment:
Can we maybe add `YAML` somewhere in the name of this constant to make its format clear?